posix/clone

```diff
@@ -115,4 +115,8 @@ esac
 
 post-fetch
 
-copy-file-content
\ No newline at end of file
+copy-file-content
+
+if [ -n "$PLUGIN_BUILD_TOOL_FILE" ]; then
```

Comment:
where is this env variable being set?

Reply:
this is set from ci-manager

Reply:
As commented in the cache plugin, let's use a flag such as `PLUGIN_ENABLE_METRICS` or similar to collect metrics - in the future there could be other details to look for.